### PR TITLE
Remedy, workorder, and common-items updated.

### PR DESCRIPTION
**Pull request by Dartellum**
Added the changes from the files that had conflicts. I also added 'What were' to common-items.

### CODE REVIEW

File: workorders.lic [hunk 1]
```diff
@@ -472,15 +477,25 @@ class WorkOrders
...
       when 'You notate', 'You put'
         bput('stow my logbook', 'You put')
       when 'The work order requires items of a higher quality'
         dispose(recipe['noun'])
-        stow_hands
+        stow_tool(left_hand)
+        stow_tool(right_hand)
       end
     end
+#    quantity.times do
```
Comment (rpherbig) [inline workorders.lic]: Is this supposed to be commented out?

**Comment by Dartellum** [comment]
Yes, can delete since I moved the dispose up under where it breaks it.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

File: remedy.lic [hunk 2]
```diff
@@ -21,7 +21,7 @@ class Remedy
       [
         { name: 'book_type', display: 'book type', options: %w(remedies), description: 'What alchemy type is this item.' },
         { name: 'chapter', regex: /\d+/i, variable: true, description: 'Chapter containing the item.' },
-        { name: 'recipe_name', display: 'recipe name', regex: /^[A-z\s\-]+$/i, variable: true, description: 'Name of the recipe, wrap in double quotes if this is multiple words.' },
```
Comment (rpherbig) [inline remedy.lic]: This looks like a merge error

File: remedy.lic [hunk 3]
```diff
@@ -271,26 +273,22 @@ class Remedy
...
+      mix("mix my #{@container} with my stick")
     when 'Upon completion you see some particulate clouding up the mixture.'
       waitrt?
       stow_item(left_hand)
       get_item('sieve')
       mix("push my #{@noun} with my sieve")
     when 'Once finished you notice clumps of material'
       waitrt?
-      mix("turn my #{@container} with my stick")
+      mix("turn my #{@container} with my mixing stick")
```
Comment (rpherbig) [inline remedy.lic]: Why is it 'stick' on line 279 and 'mixing stick' here on line 287?

**Comment by Dartellum** [comment]
Good catch. I missed that one. Stick will work.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

**Comment by Dartellum** [comment]
I am not sure it is a merge error. Does it need the minus sign in 
expression?

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

**Comment by rpherbig** [comment]
Without the '-', you cannot use any recipe with a '-' in its name. This affected forging (i.e. 'ball-peen hammer'), so I copied the change to all crafting scripts.

**Comment by Dartellum** [comment]
Okay, it was not on my local version. Do you want me to correct the PR? 
Or, can you merge then adjust this line?

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

**Comment by rpherbig** [comment]
It'd be preferable if you could correct the PR while you're making the other changes.

**Comment by Dartellum** [comment]
Okay, I think I updated everything. :)

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

File: common-items.lic [hunk 4]
```diff
@@ -34,7 +34,7 @@ module DRCI
 
   def dispose(item, trash_room = nil)
     DRCT.walk_to(trash_room) unless trash_room.nil?
-    DRC.bput("get my #{item}", 'You get')
+    DRC.bput("get my #{item}", 'You get', 'What were you')
```
Comment (rpherbig) [inline common-items.lic]: If the result of the 'get' is 'What were you', should we return early? I.e. is there any point in calling `dispose_trash`?

File: workorders.lic [hunk 5]
```diff
@@ -472,13 +477,20 @@ class WorkOrders
...
         bput("Break my #{recipe['noun']}", 'You carefully')
-        bput('stow left', 'You put')
+        /(\d+)/ =~ bput("count my first #{recipe['noun']}", 'You count out \d+ uses remaining.')
+        if Regexp.last_match(1).to_i == 5
+          bput("stow my second #{recipe['noun']}", "You put")
+        else
+          bput("stow my first #{recipe['noun']}", "You put")
+        end
         bundle_item(recipe['noun'], info['logbook'])
+        dispose(recipe['noun'])
```
Comment (rpherbig) [inline workorders.lic]: How does this work?

If you stow one of the items, and then bundle the other, when would you ever have something to dispose?

File: remedy.lic [hunk 6]
```diff
@@ -81,14 +80,15 @@ class Remedy
...
       stow_item(@herb1) # Added for when the herb is larger than 25 pieces
     end
-
+   
     if @container == 'mortar'
       get_item('pestle')
       # bput("get my pestle", "You get")
       crush("crush my #{@herb1} with my pestle")
     else
+      #bput('get my mixing stick', 'You get')
```
Comment (rpherbig) [inline remedy.lic]: Can this line be deleted?

**Comment by Dartellum** [comment]
Removed.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

**Comment by Dartellum** [comment]
Yes, returning before calling dispose_trash is a good idea. I assume it 
is something simple to do?

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

**Comment by Dartellum** [comment]
Changing the logic.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

**Comment by Dartellum** [comment]
The dispose function does a "get my #{item}," which is passed to it. 
After the bundle, it calls dispose and drops the leftover. I am also 
testing the @leftovers.times{|_| dispose(recipe['noun']) } at the end of 
the function.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

**Pull request merged**